.gitignore

```diff
@@ -15,6 +15,7 @@ __pycache__
 /results
 /output
 /scratch
+/annotations
```

Comment:
is hier een centrale plek voor? hoe verzamelen we deze?

Reply:
Dit was het mapje voor de label-studio files, die staan op mijn drive.